readme.md

```diff
@@ -111,10 +127,12 @@ try {
 
 ## API
 
-### execa(file, [arguments], [options])
+### execa(file | command, [arguments], [options])
 
 Execute a file.
 
+Arguments can be specified either inside `command` (a string) or `arguments` (an array of strings). When specified inside `command`, spaces can be escaped with a backslash. Otherwise arguments need neither escaping nor quoting.
```

Comment:
The first sentence here could be clearer and I would mention `arguments` first as I still think that should be the preferred way to use `execa`.

Reply:
I think we should also make clear that this is not a shell and you can't do stuff like `execa('echo $PATH')`.